lib/cisco_node_utils/bgp.rb

```diff
@@ -416,13 +416,24 @@ def event_history_cli
       match = config_get('bgp', 'event_history_cli', @get_args)
       if match.is_a?(Array)
         return 'false' if match[0] == 'no '
-        return 'size_' + match[1] if match[1]
```

Comment:
I believe we can simplify and remove the `Utils.nexus_evergreen?` check for `event_history_cli` like this:

```ruby
def event_history_cli
  match = config_get('bgp', 'event_history_cli', @get_args)
  if match.is_a?(Array)
    return 'false' if match[0] == 'no '
    if match[1]
      return match[1] if match[1][/\A\d+\z/]
      return 'size_' + match[1]
    end
  end
  default_event_history_cli
end
```